<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WebBrowerToolbox</class>
 <widget class="QGroupBox" name="WebBrowerToolbox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>117</width>
    <height>135</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <property name="title">
   <string>CMSWeb</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="ui_pbPreview">
     <property name="text">
      <string>Preview</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="ui_pbOpenResource">
     <property name="text">
      <string>Open Resource</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="ui_pbSettings">
     <property name="text">
      <string>Settings</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <signal>openPreview()</signal>
 </slots>
</ui>
